Tests/SymfonyIntegration/AppKernel.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\HttpKernel\Kernel;
+
+/**
+ * Appkernel for tests
+ */
+class AppKernel extends Kernel
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function registerBundles()
+    {
+        date_default_timezone_set('UTC');
+
+        $bundles = [
+            new Symfony\Bundle\FrameworkBundle\FrameworkBundle(),
+            new Symfony\Bundle\TwigBundle\TwigBundle(),
+            new PrestaShop\TranslationToolsBundle\TranslationToolsBundle(),
+        ];
+
+        return $bundles;
+    }
+
+    /**
+     * @return null
+     */
+    public function registerContainerConfiguration(LoaderInterface $loader)
+    {
+        $loader->load(__DIR__ . '/config_test.yml');
+    }
+
+    /**
+     * @return string
+     */
+    public function getCacheDir()
+    {
+        return $this->guessTempDirectoryFor('cache');
+    }
+
+    /**
+     * @return string
+     */
+    public function getLogDir()
+    {
+        return $this->guessTempDirectoryFor('logs');
+    }
+
+    private function guessTempDirectoryFor($dirname)
+    {
+        return is_writable(__DIR__ . '/../../../../build/tmp') ? __DIR__ . '/build/tmp/' . uniqid() . '/' . $dirname : sys_get_temp_dir() . '/TranslationToolsBundleTest/' . uniqid() . '/' . $dirname;
```

Comment:
issue (or expected behavior ?) of the PHPUnit Bridge : the deprecations errors are visible only once (during the build of the container in cache).

Reply:
I think it's logical. The deprecation warnings are triggered when the code in `DependencyInjection/Configuration.php` is run. This code is run at build time, when building container, and then container is dumped in cache and not rebuilt.

If we can prevent Symfony from dumping the container in cache and always loading fresh kernel, it would trigger the warnings everytime.

Reply:
this is why I've used `uniqid()` to set the path, but maybe we can disable the cache : idk